dracut/30ignition/module-setup.sh

```diff
@@ -29,9 +29,6 @@ install() {
     inst_script "$moddir/ignition-setup.sh" \
         "/usr/sbin/ignition-setup"
 
-#   inst_script "$moddir/retry-umount.sh" \
-#       "/usr/sbin/retry-umount"
```

Comment:
I didn't mention this in the commit message, but we can drop this completely too now since we're going with the `MountFlags=slave` approach.